tools/cp-caps/rcp_caps_test.py

```diff
@@ -85,112 +92,92 @@ def test_diag_commands(self):
         self.__ref.diag_stop()
         self.__dut.diag_stop()
 
+    class Frame:
+
```

Comment:
Maybe helpful to document the attributes.

Also, what about using `@dataclass`?